log/sentry.go

```diff
@@ -71,12 +75,33 @@ func InitSentry(cfg SentryConfig) (io.Closer, error) {
 	if cfg.SampleRate != nil && *cfg.SampleRate >= 0 && *cfg.SampleRate <= 1 {
 		sampleRate = *cfg.SampleRate
 	}
+
+	// Improve legibility of Sentry errors by using the error message as header
+	// instead of the error type and marking stack trace frame from
+	// baseplate.go as not in-app.
+	const base = "github.com/reddit/baseplate.go"
+	beforeSend := func(event *sentry.Event, hint *sentry.EventHint) *sentry.Event {
+		for i, exception := range event.Exception {
+			event.Exception[i].Type, event.Exception[i].Value = event.Exception[i].Value, event.Exception[i].Type
+			for j, frame := range exception.Stacktrace.Frames {
+				if frame.Module == base || strings.HasPrefix(frame.Module, base+"/") {
```

Comment:
super nit: we probably want to define `const prefix = base + "/"` as well, as doing the concatenation in every for loop could be wasteful.